client/service/src/metrics.rs

```diff
@@ -138,16 +152,27 @@ impl MetricsService {
 		txpool_status: &PoolStatus,
 		net_status: &NetworkStatus<T>,
 	) {
+		let now = Instant::now();
+
 		let best_number = info.chain.best_number.saturated_into::<u64>();
 		let best_hash = info.chain.best_hash;
 		let num_peers = net_status.num_connected_peers;
 		let finalized_number: u64 = info.chain.finalized_number.saturated_into::<u64>();
-		let bandwidth_download = net_status.average_download_per_sec;
-		let bandwidth_upload = net_status.average_upload_per_sec;
+		let total_bytes_inbound = net_status.total_bytes_inbound;
+		let total_bytes_outbound = net_status.total_bytes_outbound;
 		let best_seen_block = net_status
 			.best_seen_block
 			.map(|num: NumberFor<T>| num.unique_saturated_into() as u64);
 
+		let elapsed = (now - self.last_update).as_secs();
+		let inbound_per_sec = (total_bytes_inbound - self.last_total_bytes_inbound) / elapsed;
+		let outbound_per_sec = (total_bytes_outbound - self.last_total_bytes_outbound) / elapsed;
```

Comment:
If `elapsed` = 0, there will be a panic.

Reply:
Addressed in https://github.com/paritytech/substrate/pull/6870/commits/6c96875d2a77371fb323eaa5a5f62ac66fdb05cf.